clients/ui/frontend/src/__tests__/unit/testUtils/hooks.ts

```diff
@@ -7,14 +7,8 @@ import {
 } from '@testing-library/react';
 import { queries, Queries } from '@testing-library/dom';
 
-export type BooleanValues<T> = T extends
-  | boolean
-  | number
-  | string
-  | null
-  | undefined
-  // eslint-disable-next-line @typescript-eslint/ban-types
-  | Function
+// eslint-disable-next-line @typescript-eslint/no-unsafe-function-type
```

Comment:
as this is a test, I just ignore the warning. @Griffin-Sullivan please let me know if we should refactor it instead.

Reply:
fine with me